### ISSUE TITLE
spriteNamespace and namespace should allow empty string as value

### ISSUE THREAD

**Issue by didoo** [frontainer/gulp-sprite-glue#5]
At the line
https://github.com/frontainer/gulp-sprite-glue/blob/master/index.js#L117
you are checking that the option is not `false` as negation of the value set; but on this case, is a user wants to add an empty string as value for the `spriteNamespace` and `namespace` options, the test results as true, and the script falls back to the predefined option. In this case, the option is not set, and the gulp library use the default values (which are not empty strings).
Simply put: to output this command:

```
glue [path_to_images] --scss --recursive --namespace=sprite --sprite-namespace= --img=build/dev/images/ --css=build/dev/sass
```

you have to set the option in this way:

```
{ namespace: ' ', spriteNamespace: ' ' }
```

that is, using spaces.
